spring-cloud-skipper/spring-cloud-skipper-server-core/pom.xml

```diff
@@ -20,6 +20,34 @@
 	</properties>
 
 	<dependencies>
+		<dependency>
```

Comment:
We should not need all of these dependencies. The ones that are pulled in will have their version number adjusted according to the dep. mgmt. - again... only need to set a property.